README.md

```diff
@@ -1,5 +1,65 @@
-# Randomly generated 3 x 300 patient data
+#Randomly generated 3 x 300 patient data
```

Comment:
a space is needed after `#` for markdown to recognize this as a title, same applies to all of them.